tests/ui_json/forms_test.py

```diff
@@ -448,7 +553,7 @@ def test_flatten():
     )
     assert str(form.flatten()) == data_uid
 
-    form = DataForm(
+    form = DataOrValueForm(

```

Comment:
you don't have to do it now, but for repetitive test like this, you can use pystet mixtures ;-)